README.rst

```diff
@@ -46,3 +46,233 @@ Sample results on macOS with clang and libc++:
+	8     25
+	9      0
+	10      1
+	2020-06-21T10:25:13-04:00
+	Running ../file-int-benchmark
+	Run on (12 X 3593.26 MHz CPU s)
+	CPU Caches:
+	L1 Data 32 KiB (x6)
+	L1 Instruction 32 KiB (x6)
+	L2 Unified 512 KiB (x6)
+	L3 Unified 16384 KiB (x1)
+	Load Average: 0.00, 0.04, 0.23
+	----------------------------------------------------------------
+	Benchmark                      Time             CPU   Iterations
+	----------------------------------------------------------------
+	fprintf                 64107364 ns     64106119 ns           11
+	std_ofstream            60662327 ns     60662397 ns           11
+	fmt_print               50619350 ns     50618835 ns           14
+	std_to_chars            13745430 ns     13745453 ns           50
+	fmt_to_string           19279941 ns     19279726 ns           37
+	fmt_format_runtime      41865575 ns     41864968 ns           16
+	fmt_format_compile      22275384 ns     22275375 ns           32
+	fmt_format_to_runtime   28825158 ns     28824894 ns           24
+	fmt_format_to_compile   14296560 ns     14296560 ns           48
+	fmt_format_int          13562310 ns     13562309 ns           51
+	boost_lexical_cast      33195733 ns     33174949 ns           21
+	boost_format           241966400 ns    241964154 ns            3
+	boost_karma_generate    14441921 ns     14441750 ns           48
+	voigt_itostr            20043046 ns     20042388 ns           35
+	u2985907                13534979 ns     13534671 ns           52
+	u2985907_correct        10393649 ns     10388901 ns           67
+	std_to_chars_fast       10851778 ns     10851776 ns           64
+	decimal_from            14454558 ns     14454380 ns           48
+	stout_ltoa              22757248 ns     22757236 ns           31
+	fast_io_concat          14376316 ns     14376320 ns           49
+	fast_io_concatln        14233867 ns     14233865 ns           48
+	fast_io_print_reserve   13492135 ns     13492138 ns           51
+	fast_io_println         10339622 ns     10339623 ns           67
+
+concat benchmark
+
+.. code::
+	cqwrteur@DESKTOP-C4VAUFM:~/format-benchmark/build$ ./concat-benchmark
+	2020-06-21T08:49:23-04:00
+	Running ./concat-benchmark
+	Run on (12 X 3593.26 MHz CPU s)
+	CPU Caches:
+	L1 Data 32 KiB (x6)
+	L1 Instruction 32 KiB (x6)
+	L2 Unified 512 KiB (x6)
+	L3 Unified 16384 KiB (x1)
+	Load Average: 1.65, 1.57, 0.68
+	------------------------------------------------------------
+	Benchmark                  Time             CPU   Iterations
+	------------------------------------------------------------
+	naive                   87.3 ns         87.3 ns      7980351
+	append                  61.1 ns         61.1 ns     11319732
+	appendWithReserve       41.8 ns         41.8 ns     16762382
+	format_compile          74.8 ns         74.8 ns      9358126
+	format_runtime           112 ns          112 ns      6235314
+	format_to               84.3 ns         84.3 ns      8146667
+	fast_io_print           23.3 ns         23.3 ns     30091614
```

Comment:
This makes no sense because the whole purpose of benchmark is `std::string` concat, not producing some random data structures.

Reply:
> This makes no sense because the whole purpose of benchmark is `std::string` concat, not producing some random data structures.

Oh you do understand your benchmark makes no sense? Then why format_int makes sense when it does not generate std::string?

Reply:
I'm referring to `std::string` concatenation benchmark. Integer formatting has nothing to do with `std::string`.

Reply:
> I'm referring to `std::string` concatenation benchmark. Integer formatting has nothing to do with `std::string`.

fast_io::concat does both concating and integer formatting to std::string why do you compare it to u2985907?

Reply:
There is no integer formatting here. This is a **string concatenation** benchmark. Can you even read the code that you've submitted?